data/apache/000-default.conf

```diff
@@ -0,0 +1,12 @@
+<VirtualHost *:80>
+    <Location /assets>
+         ProxyPass http://imgproxy:8080
```

Comment:
this needs to be changed to the consul(?)

Reply:
No, needs to be changed in in de deploy-cms.yml file, so it links to the imgproxy